deploy/operator.yaml

```diff
@@ -44,16 +44,16 @@ spec:
             - name: ZYNC_IMAGE
               value: "quay.io/3scale/zync:nightly"
             - name: SYSTEM_MEMCACHED_IMAGE
-              value: "memcached:1.5"
+              value: "registry.stage.redhat.io/3scale-amp2/memcached-rhel7:3scale2.8"
             - name: BACKEND_REDIS_IMAGE
-              value: "centos/redis-32-centos7"
+              value: "registry.redhat.io/rhscl/redis-32-rhel7:3.2"
             - name: SYSTEM_REDIS_IMAGE
-              value: "centos/redis-32-centos7"
+              value: "registry.redhat.io/rhscl/redis-32-rhel7:3.2"
             - name: SYSTEM_MYSQL_IMAGE
-              value: "centos/mysql-57-centos7"
+              value: "registry.redhat.io/rhscl/mysql-57-rhel7:5.7"
             - name: SYSTEM_POSTGRESQL_IMAGE
-              value: "centos/postgresql-10-centos7"
+              value: "registry.redhat.io/rhscl/postgresql-10-rhel7"
             - name: ZYNC_POSTGRESQL_IMAGE
-              value: "centos/postgresql-10-centos7"
+              value: "registry.redhat.io/rhscl/postgresql-10-rhel7"
             - name: OC_CLI_IMAGE
-              value: "quay.io/openshift/origin-cli:4.2"
+              value: "registry.redhat.io/openshift4/ose-cli"
```

Comment:
:latest tag of openshift4/ose-cli has not been tested. We should use ose-cli:4.2 if we want to be safe

Reply:
:+1:

Reply:
It is strongly recommended to update to 4.4

`Updated image available
Red Hat strongly recommends updating to the newest image version 
unless otherwise defined by the support policy of Red Hat, Inc.`

https://catalog.redhat.com/software/containers/openshift4/ose-cli/5cd9ba3f5a13467289f4d51d?tag=v4.2&container-tabs=gti